tigerpath/majors_and_certificates/majors/GER_2018.json

```
@@ -0,0 +1,403 @@
+          "req_list": [
+            {
+              "name": "German",
+              "max_counted": 1,
+              "min_needed": 5,
+              "explanation": "Consists of five courses in the German Department (usually no more than two at the 200 level).",
+              "req_list": [
+                {
+                  "name": null,
+                  "max_counted": null,
+                  "min_needed": null,
+                  "explanation": null,
+                  "course_list": [
+                    "GER 3*",
+                    "GER 4*",
+                    "GER 5*"
+                  ]
+                },
+                {
+                  "name": null,
+                  "max_counted": 2,
+                  "min_needed": null,
+                  "explanation": null,
+                  "course_list": [
+                    "GER 2*"
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "Cognates",
+              "max_counted": 1,
+              "min_needed": 3,
+              "explanation": "<i>Editor's Note: Most of these courses fall under this requirement but as TigerPath does not verify this.</i>\n\nThree upper-level courses in a second literature.",
+              "course_list": [
+                "LANG 3*",
+                "LANG 4*",
+                "LANG 5*",
+                "ENG 3*",
+                "ENG 4*",
+                "ENG 5*"
+              ],
+              "excluded_course_list": [
+                "GER *"
+              ]
+            }
+          ]
+        },
+        {
+          "name": "German Culture and Politics",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": "Students may combine a concentration in German literature and culture with a concentration in German/European politics and political theory. Four courses will be taken in the German Department (usually no more than two at the 200 level), on topics pertaining to German literature and culture, and four courses will be taken in the Department of Politics (usually no more than two at the 200 level), on topics pertaining to German/European politics or political theory. The senior thesis may focus on any political topic with a substantive German-related component. Upon graduation, a letter will be issued by the department certifying the completion of a program in German cultural studies with a concentration in politics.",
+          "req_list": [
+            {
+              "name": "German",
+              "max_counted": 1,
+              "min_needed": 4,
+              "explanation": "Four courses will be taken in the German Department (usually no more than two at the 200 level), on topics pertaining to German literature and culture.",

```

The [handbook](https://german.princeton.edu/sites/default/files/2019-11/2019-2020%20current%20Undergraduate%20Handbook%20Nov.pdf) provides some specific course recommendations, so why not provide the info. 
In fact we might consider switching to instead having a course list with these specific courses, but I guess it's also fine as is.
```suggestion
              "explanation": "Four courses will be taken in the German Department (usually no more than two at the 200 level), on topics pertaining to German literature and culture. Recommended departmental courses are GER 207, 208, 211, 306, 307, 309, and 324.",
```